src/App.js

```diff
@@ -10,7 +14,21 @@ export default class App extends Component {
   render() {
     return (
       <div className="App">
-        <Calculator />
+        <Navigation />
+        <Routes>
+          <Route
+            path="/math-magicians/"
+            element={(<Home />)}
+          />
+          <Route
+            path="/math-magicians/calculator"
+            element={(<Calculator />)}
+          />
+          <Route
+            path="/math-magicians/quote"
```

Comment:
- [ ] **[OPTIONAL]** I think we can remove the "/math-magicians" sections for these pages, like so:

  ```
  path="/"
  ...
  path="/calculator"
  ...
  path="/quote"
  ```
  And also change the paths in the navigation component.
  ```
  path: '/',
  ...
  path: '/calculator',
  ...
  path: '/quote',
  ```

Reply:
Thank you for your suggestion.🙂
I think it is important for deployment because when you deploy your page it is not showing the home page.
If you know another solution, please share it with me👍